lib/active_scaffold/helpers/form_column_helpers.rb

```diff
@@ -197,11 +197,11 @@ def form_attribute(column, record, scope = nil, only_value = false, col_class =
         end
         if field
           field << loading_indicator_tag(:action => :render_field, :id => params[:id]) if column.update_columns
-          field << content_tag(:span, column.description(record, scope), :class => 'description') if column.description.present?
+          field << content_tag(:span, column.description(record, scope), :class => 'description') if column.description(record, scope).present?
```

Comment:
You could use a local variable to avoid calling column.description(record, scope) twice